firefox-ios/Client/Frontend/Widgets/OneLineTableViewCell.swift

```diff
@@ -36,6 +37,15 @@ class OneLineTableViewCell: UITableViewCell,
         static let cornerRadius: CGFloat = 5
     }
 
+    var reorderControlImageView: UIImageView? {
+            let reorderControl = self.subviews.first { view in
+                view.classForCoder.description() == "UITableViewCellReorderControl"
```

Comment:
`"UITableViewCellReorderControl"` is internal, I think we should find another way for this. Relying on it might make the code prone to breaking with future updates to iOS. If it implies having a new `UITableViewCell` custom class so be it.

Reply:
After discussion, it seems this solution is simplest and is the way to go 👍